### PR TITLE
Add AlmaLinux 9

### PR DESCRIPTION
**Pull request by ochaplashkin**
AlmaLinux 9 based on the Red Hat Enterprise Linux 9 sources.

### CODE REVIEW

File: almalinux/9/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,24 @@
+FROM almalinux:9
+MAINTAINER Oleg Chaplashkin <ochaplashkin@tarantool.org>
+
+# Enable extra tools
+RUN dnf -y install wget yum-utils
+
+# Enable extra repositories
+RUN dnf -y install epel-release
```
Comment (ylobankov) [inline almalinux/9/Dockerfile]: Due to Docker architecture, this step as a separate one doesn't make big sense. One of the best practices for creating images states:

Try to reduce the number of layers in your image by minimizing the number of separate `RUN` commands in your Dockerfile. You can do this by consolidating multiple commands into a single `RUN` line and using your shell's mechanisms to combine them together.

File: almalinux/9/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+RUN dnf -y group install 'Development Tools'
+RUN dnf -y install \
+    cmake \
+    sudo
+
+# Enable sudo without tty
+RUN sed -i.bak -n -e '/^Defaults.*requiretty/ { s/^/# /;};/^%wheel.*ALL$/ { s/^/# / ;} ;/^#.*wheel.*NOPASSWD/ { s/^#[ ]*//;};p' /etc/sudoers
+
+# Cleanup DNF metadata and decrease image size
+RUN dnf clean all
```
Comment (ylobankov) [inline almalinux/9/Dockerfile]: Considering my aforementioned comment, the resulting Dockerfile may look something like this:

```Dockerfile
FROM almalinux:9
MAINTAINER Oleg Chaplashkin <ochaplashkin@tarantool.org>

RUN dnf -y install \
        # Install auxiliary tools.
        sudo \
        wget \
        yum-utils \
        # Install extra repositories.
        epel-release && \
    # Install base toolset.
    dnf -y group install 'Development Tools' && \
    dnf -y install cmake && \
    # Enable CRB (CodeReady Builder) - the repository containing
    # additional packages for developers. CRB is an equivalent of
    # PowerTools for CentOS 7/8 and AlmaLinux 8.
    dnf config-manager --set-enabled crb && \
    # Cleanup DNF metadata to decrease image size.
    dnf clean all && \
    rm -rf /var/cache/dnf

# Enable sudo without tty.
RUN sed -i.bak -n -e '/^Defaults.*requiretty/ { s/^/# /;};/^%wheel.*ALL$/ { s/^/# / ;} ;/^#.*wheel.*NOPASSWD/ { s/^#[ ]*//;};p' /etc/sudoers
```

Try to build an image with this Dockerfile and you will see the difference in the image size.

Comment (ochaplashkin) [reply almalinux/9/Dockerfile]: I tried your image and got the following:

Size: 1.46GB -> 1.3GB (decreased)
Build time (with my network): ~120 sec. (remains unchanged)

OK, I fixed my image in the next commit. Thanks!

File: almalinux/9/Dockerfile (hunk 1 above)
Comment (ochaplashkin) [reply almalinux/9/Dockerfile]: Fixed.

**Review by ylobankov — APPROVED** [review]
No objections from my side anymore.

**Review by Totktonada — APPROVED** [review]
OK for me.

**Pull request merged**